terraform/github/variables.tf

```diff
@@ -24,14 +24,14 @@ locals {
       name        = "server"
       description = "REST API for benchttp."
     },
-    "webapp" = {
-      name        = "webapp"
-      description = "Front-end application for benchttp users."
-    },
     "testing_server" = {
```

Comment:
Isn't it `cobaye` now ?

Reply:
Fixed in https://github.com/benchttp/infra/pull/7!